bot/management/commands/bot.py

```diff
@@ -26,46 +26,67 @@ async def repost(update, context):
             inline_keyboard = update.channel_post.reply_markup
         else:
             work_content = None
+            inline_keyboard = update.channel_post.reply_markup
```

Comment:
Why do not just write it under if-else block? Line is the same for all cases.

Reply:
In one place it is called inline_keyboard, in another you call it markup.
Should be update_markup and markup. Otherwise I think these are 2 different things. It does not affect if code is working, but very confusing for the readability.